1.9/usage/cli/command-reference/dcos-task/dcos-task-exec.md

```diff
@@ -0,0 +1,34 @@
+---
+post_title: dcos task exec
+menu_order: 0
+---
+
+# Description
+Launch a process inside of a task container.
+
+# Usage
+
+```bash
+dcos task exec [OPTION]
+```
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--interactive, -i`   |             |  Attach a STDIN stream to the remote command for an interactive session. |
+| `--tty, -t`   |             |  Attach a TTY to the remote stream. |
+
+# Positional arguments
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `<args>`   |             |  Additional arguments to pass to the command. |
```

Comment:
Can you order these in the same order they appear on the command line?
Also, it may be useful to borrow some of the text from here: https://github.com/dcos/dcos-cli/blob/master/cli/dcoscli/data/help/task_exec.txt